### PR TITLE
Mailing

### PR DESCRIPTION
**Pull request by ruxmur**

### CODE REVIEW

File: app/Http/Controllers/ContactUsController.php [hunk 1]
```diff
@@ -2,12 +2,41 @@
...
+    {
+
+        // dd($request->validated());
+        $data = $request->validated();
+        \Log::debug('test', $data);
+
+        \Mail::send(
+            'emails.contactUs',
+            [
+                'firstname' => $data['firstname'],
```
Comment (egorbwork) [inline app/Http/Controllers/ContactUsController.php:26]: At lesson I requested, to have at least one select on the form.

File: app/Http/Controllers/ContactUsController.php [hunk 2]
```diff
@@ -2,12 +2,41 @@
...
+                'lastname' => $data['lastname'],
+                'email' => $data['email'],
+
+                'messageText' => $data['message'],
+
+            ],
+            function (Message $message) use ($data) {
+                $message->subject('message from ' . $data['email']);
+                $message->to('tech@baloon.app');
```
Comment (egorbwork) [inline app/Http/Controllers/ContactUsController.php:35]: please adjust to and from for your project values.

File: app/Http/Controllers/ContactUsController.php (hunk 1 above)
Comment (egorbwork) [reply app/Http/Controllers/ContactUsController.php:26]: looks that select exists but it is not used in mail.

**Review by egorbwork — CHANGES_REQUESTED** [review]
A bit incomplete but looks good.

File: app/Http/Controllers/ContactUsController.php (hunk 1 above)
Comment (ruxmur) [reply app/Http/Controllers/ContactUsController.php:26]: added missing elements, but the changes are on  Mailing-dependency-injection branch

File: app/Http/Controllers/ContactUsController.php (hunk 2 above)
Comment (ruxmur) [reply app/Http/Controllers/ContactUsController.php:35]: changed

**Review by egorbwork — APPROVED** [review]
Ok, will take a look what was done in next pr.

**Pull request merged**